aiida/manage/configuration/options.py

```diff
@@ -30,26 +31,27 @@ def name(self) -> str:
         return self._name
 
     @property
-    def schema(self) -> Dict[str, Any]:
-        return self._schema
+    def valid_type(self) -> Any:
+        return self._field.type_
 
     @property
-    def valid_type(self) -> Any:
-        return self._schema.get('type', None)
+    def schema(self) -> Dict[str, Any]:
+        return self._schema
 
     @property
     def default(self) -> Any:
-        return self._schema.get('default', None)
+        return self._field.default
 
     @property
     def description(self) -> str:
-        return self._schema.get('description', '')
+        return self._field.field_info.description
 
     @property
     def global_only(self) -> bool:
-        return self._schema.get('global_only', False)
+        from .config import ProfileOptionsSchema
```

Comment:
I don't understand why this import is called from method, will this improve the performance? Same for `GlobalOptionsSchema` below.

Reply:
It is necessary, otherwise there are circular imports and `pydantic` will except.